.travis.yml

```diff
@@ -39,4 +39,5 @@ script:
     - ./test.sh --nologcapture
     - ./lint.sh
     - ../node_modules/.bin/mocha static/build_test.js
+    - docker run --rm=true -i -w "/go/src/k8s.io/test-infra" -v "${GOPATH}/src/k8s.io/test-infra:/go/src/k8s.io/test-infra" 'gcr.io/google_containers/kubekins-job-builder:5' bash -c 'jenkins-jobs --ignore-cache test -o output jenkins/job-configs:jenkins/job-configs/kubernetes-jenkins'
     - go test k8s.io/test-infra/testgrid/config/...
```

Comment:
I am not thrilled about the fact that go test will now fail unless I have a magically created output file. Will you make sure that `go test k8s.io/test-infra/testgrid/config/...` works on a clean checkout?

Two possible ways to make that happen:
1. Move the `docker run` command inside `jenkinsvalidate_test.go` so the test creates its own output.
2. Change `jenkinsvalidate_test.go` to a program instead of a unit test and invoke it directly via `go run` instead of `go test`
3. Both of the above
4. Some other alternative you come up with.